### PR TITLE
Issue 661: Use 'Sex Not Entered' for patients with missing gender on …

### LINKED ISSUE #661: Patients without identified sex show up as null or undefined on patient reports
**Expected behavior:**
When running patient reports such as admission report which groups counts by sex, those patients without a sex filled out in the demographic information show up as "Sex Not Entered" or a similar label
**Actual behavior:**
In patient reports such as admission report which groups counts by sex, patients without a sex filled out in the demographic information show up as null or undefined
**Steps to reproduce:**
1. Create a new patient, but leave the Sex field empty.
2. Admit that new patient
3. Run the admissions detail patient report
**Screenshots (if applicable):**
screen shot 2016-09-21 at 12 54 36 pm

**OS and Browser:**
All

### PR DESCRIPTION
**Pull request by akong**
Fixes #661 .

**Changes proposed in this pull request:**
- Use 'Sex Not Entered' for patients without gender entered for reports.

cc @HospitalRun/core-maintainers

### CODE REVIEW

File: app/patients/reports/controller.js [hunk 1]
```diff
@@ -568,13 +568,17 @@ export default AbstractReportController.extend(PatientDiagnosis, PatientVisits,
           admissionDate: visit.get('startDate'),
           dischargeDate: visit.get('endDate')
         };
-        var sexGrouping = patientBySex[visit.get('patient.sex')];
+        var sex = visit.get('patient.sex');
+        if (!sex) {
+          sex = 'Sex Not Entered';
```
Comment (jkleinsc) [inline app/patients/reports/controller.js]: @akong Can you localize the text here?  You can put the label under patient.labels.sexNotEntered in the localization file at app/locales/en/translations.js and then you can pull it in to the controller by calling
```javascript
this.get('i18n').t('patients.labels.sexNotEntered'),
```

Comment (akong) [inline app/patients/reports/controller.js]: Got it!  Will update the PR later today.

**Comment by tangollama** [comment]
@jkleinsc any idea why greenkeeper is not reporting a status? I'm inclined to merge the change, but I don't want to break our established process.

File: app/patients/reports/controller.js (hunk 1 above)
Comment (akong) [inline app/patients/reports/controller.js]: @jkleinsc Updated the pull request with the suggested changes.

**Pull request merged**